web/router.ex

```diff
@@ -11,9 +11,7 @@ defmodule RemoteRetro.Router do
     plug :fetch_flash
     plug :protect_from_forgery
     plug :put_secure_browser_headers
-    if Application.get_env(:remote_retro, :add_mock_user_to_session) do
-      plug :add_mock_user_to_session
-    end
+  
```

Comment:
@vanderhoop We got it working on localhost without doing anything like that.  If you test it out with the credentials I pasted in slack you can see for yourself.